tests/testthat/_snaps/repair.md

```diff
@@ -0,0 +1,21 @@
+# repair() can update DESCRIPTION files for GitHub packages
+
+    Code
+      . <- repair()
+    Output
+      - No issues found with the project library's cache links.
```

Comment:
I sort of wonder if I should print little headers between each section.

Reply:
Yeah, maybe like:

```
# Library cache links
- No problems found
```

?